src/app/models/parsed-elements.ts

```diff
@@ -15,6 +15,10 @@ export type HTMLData = GenericElementData & {
 export interface TextData extends GenericElementData {
     text: string;
 }
+export type NoteType = 'critical' | 'comment';
```

Comment:
Are these fixed or may change by configuration?

Reply:
They're fixed

Reply:
If we move the layout logic inside the note component (to output differently notes in footer or in entities) this dual distinction between critical and comment is no more valid. Moreover if we have a look at the TEI guidelines there could be assigned with a very generic string.